spec/status_spec.rb

```diff
@@ -403,30 +403,42 @@
   end
 
   describe 'errors' do
-    it 'lets you add an error' do
-      status = make_status(state: :started)
-      status.add_error({'message' => 'blah', 'foo' => 2})
-      expect(status.errors).to eq [{'message' => 'blah', 'foo' => 2}]
+    let(:status) { make_status(state: :started) }
+
+    describe 'adding' do
```

Comment:
should this be describe '#add_error'?   it's a test for the add_error instance method on the status object right?

Reply:
yep, good idea